docs/UserGuide.md

```diff
@@ -139,57 +139,69 @@ Company at specified index removed and list of companies updated
 </br>
 If INDEX is out of bounds:
 </br>
-`Index exceeds the number of contacts!`
+`The company index provided is invalid`
 
 If INDEX is zero or negative:
 </br>
-`Index must be positive!`
+`Invalid command format`
```

Comment:
maybe can consider retaining specific error messages instead of using "invalid" for better user experience. applies to rest of the code below as well.

Reply:
I think good practice to give them an example of a correct input, because they may not know what <index> refers to.